app/lib/actions/pulp/orchestration/orphan_cleanup/remove_orphans.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Pulp
+    module Orchestration
+      module OrphanCleanup
+        class RemoveOrphans < Pulp::Abstract
+          def plan(proxy)
+            plan_action(Actions::Pulp::OrphanCleanup::RemoveUnneededRepos, proxy) unless proxy.pulp_master?
+            plan_self(:capsule_id => proxy.id)
+          end
+
+          def run
+            pulp_resources.content.remove_orphans
+          end
```

Comment:
as per our chat, we need to either make this an abstract_async_task (changing run to 'invoke_external_task') or move this to another task that does.